presidio-analyzer/tests/test_analyzer_engine.py

```diff
@@ -556,3 +560,32 @@ def test_given_interpretability_requested_then_response_contains_analysis(
 
     assert len(results) == 1
     assert results[0].analysis_explanation is not None
+
+
+def test_read_test_nlp_conf_file_returns_spacy_nlp_engine(
```

Comment:
how about also testing a not supported nlp engine?

Reply:
added more tests